src/commands/plugins.rs

```diff
@@ -474,6 +538,26 @@ impl PluginDescriptor {
     }
 }
 
+impl std::fmt::Display for PluginDescriptor {
```

Comment:
I'd be a little bit cautious about giving this as _the_ display format for a plugin descriptor.  `List::print`, for example, prints them in its own specific format.

My suggestion for the multiselect display would be either:

* Just the plugin name (e.g. `foo`)
* The plugin name with the current version and what it would upgrade to (e.g. `foo (from 1.0.0 to 1.2.0)`)

You can use strings rather than descriptors in the `Multiselect` `items` list as long as you ensure they keep the same order (have the same indexes) so that indexing back into your descriptor list still works e.g.

```
// using the "just the name" example
let names: Vec<_> = descriptors.iter().map(|d| d.name.to_string()).collect();  // preserves order
let indexes = dialoguer::Multiselect::new().items(&names)...;
let selected = elements_at(descriptors, indexes);
```

Reply:
I combined the two ideas you gave me and create a `names` vector, and using the macro `format!` I appended the versions to the name to display more information to the user as you suggested.